### ISSUE TITLE
API: Add ability to control client dimming

### ISSUE THREAD

**Issue by knitehawk** [xibosignage/xibo#52]
it would be nice to have the ability to control xibo display dimming directly.
this could be a variable in the API controlling screen render Alpha or via an over-lay black screen alpha

**Comment by dasgarner** [comment]
We've experimented with whole screen overlays that have an alpha value, which does appear to work - although we haven't tested for any duration.

Does HDMI-CEC have support for adjusting screen brightness?

**Comment by reemaRaven** [comment]
Hi,

Has this enhancement been added?
Eagerly waiting on this one!

Thanks,
Reema

**Comment by dasgarner** [comment]
You can control brightness and contrast from the CMS using our webOS player and compatible monitor. We don't have any controls available for that through HDMI-CEC, apart from if you use your own Commands to achieve that.
